torch/csrc/distributed/rpc/process_group_agent.cpp

```diff
@@ -135,7 +135,7 @@ ProcessGroupAgent::ProcessGroupAgent(
   }
 
   allWorkerInfo_.reserve(pg_->getSize());
-  for (int rank = 0; rank < (int)tmpWorkerIds.size(); ++rank) {
+  for (worker_id_t rank = 0; rank < tmpWorkerIds.size(); ++rank) {
```

Comment:
can we fix this clang-tidy issue? although I guess the `tmpWorkerIds` is valid when looping, but would like to ensure there's no overflow.